Goldtree/Goldtree/Program.cs

```diff
@@ -38,33 +35,33 @@ public void Log(string Text, bool NewLine = true)
             Console.ForegroundColor = prev;
             Console.Write(" ");
             Console.Write(Text);
-            if(NewLine) Console.WriteLine();
-            if(Close)
-            {
-                if(Program.USB != null)
-                {
-                    Command c = new Command(CommandId.Finish);
-                    Program.USB.Write(c);
-                }
-                Console.WriteLine("Press any key to exit...");
-                Console.ReadKey();
-                Environment.Exit(1);
-            }
         }
     }
 
-    public class Program
+    public class GoldTree
     {
         public static readonly string Name = "Goldtree";
         public static readonly string Description = "Goldleaf's USB installation client";
 
-        public static readonly LogMode Log = new LogMode("Log", ConsoleColor.Cyan);
-        public static readonly LogMode Warn = new LogMode("Warn", ConsoleColor.Yellow);
-        public static readonly LogMode Error = new LogMode("Error", ConsoleColor.Red, true);
+        public readonly LogMode Log = new LogMode("Log", ConsoleColor.Cyan);
+        public readonly LogMode Warn = new LogMode("Warn", ConsoleColor.Yellow);
+        public readonly LogMode Error = new LogMode("Error", ConsoleColor.Red);
+
+        private UsbK USB;
 
-        public static UsbK USB = null;
+        public GoldTree()
```

Comment:
Move all this code out of the constructor and into a `void Main(string[] args)` (see bottom of the file for additional feedback).